libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -341,11 +343,18 @@ private String getSecretValue(String criId, String connection, String secretId)
     }
 
     private <T> T getCriConfigForType(String connection, String criId, Class<T> configType) {
+        return getCriConfigForType(connection, criId, configType, true);
```

Comment:
Should we update this (and any other methods that take `credentialIssuerId` as an argument) to use the enum type instead?

Reply:
I started doing this and its resulted in lots more changes required! Might be worth tackling in a separate PR

Reply:
https://govukverify.atlassian.net/browse/PYIC-6816

Reply:
Thanks!